### PR TITLE
Newsletter 7: Rhea, Oxidator, Dwarfs, Univer, Cat

### PR DESCRIPTION
**Pull request by ozkriff**
_Part of #64 ("Newsletter 7: Coordination/Tracking #64")_

@Ruddle @Thinkofname @Fryer, r?

### CODE REVIEW

File: content/posts/newsletter-007/index.md [hunk 1]
```diff
@@ -238,6 +238,126 @@ network play][designing_network_play].
...
+
+[![Map editor demo: use pencil tool to instantly create a lake and mountains](oxidator-map-editor.gif)][oxidator-video-map-editor]
+
+^ _Demo of the map editor_
+
+[oxidator]: https://github.com/Ruddle/oxidator
+[@Ruddle]: https://github.com/Ruddle
+[oxidator-video-play]: https://streamable.com/499j0
+[oxidator-video-unit-editor]: https://streamable.com/ywr44
+[oxidator-video-map-editor]: https://streamable.com/e5zdj
```
Comment (Uriopass) [inline content/posts/newsletter-007/index.md]: Link is dead

Comment (Uriopass) [reply content/posts/newsletter-007/index.md]: Could link to the gif instead: https://github.com/Ruddle/oxidator/raw/master/etc/map_editor.gif

File: content/posts/newsletter-007/index.md [hunk 2]
```diff
@@ -238,6 +238,126 @@ network play][designing_network_play].
...
+> and will have to be copied from the game on Steam.
+
+_Discussions:
+[/r/rust](https://reddit.com/r/rust/comments/f5fiii/univercity_open_source_release_university)_
+
+[univer-1-0-opensource]: https://steamcommunity.com/gid/103582791461907043/announcements/detail/1694978169192631655
+[univer-steam]: https://store.steampowered.com/app/808160/UniverCity
+[univer-source]: https://github.com/Thinkofname/UniverCity
+
+### [Dwarf World][dwarf-world]
```
Comment (AlexEne) [inline content/posts/newsletter-007/index.md]: Oh gosh, I didn't think this is worth mentioning. Thanks :D

**Comment by ozkriff** [comment]
:timer_clock: Merging in a few hours if there're no objections.

**Pull request merged**